br/backup-and-restore-overview.md

```diff
@@ -0,0 +1,123 @@
+---
+title: BR Overview
+summary: Learn about the definition and functions of BR.
+aliases: ['/docs/dev/br/backup-and-restore-tool/','/docs/dev/reference/tools/br/br/','/docs/dev/how-to/maintain/backup-and-restore/br/','/tidb/dev/backup-and-restore-tool']
+---
+
+# BR Overview
+
+[BR](https://github.com/pingcap/br) (Backup & Restore) is a command-line tool for **distributed backup and restoration** of the TiDB cluster data. In addition to regular backup and restoration, you can also use BR for large-scale data migration as long as compatibility is ensured.
+
+This document describes BR's architecture, functions, and usage tips.
+
+## BR architecture
+
+BR sends a backup or restoration command to each TiKV node. After receiving the command, TiKV performs the corresponding backup or restoration operation.
+
+Each TiKV node has a path in which the backup files generated in the backup operation are stored and from which the stored backup files are read during the restoration.
+
+![br-arch](/media/br-arch.png)
+
+For detailed information about the BR design, see [BR Design Principles](/br/backup-and-restore-design.md).
+
+## BR features
+
+This section describes BR features and performance indicators.
+
+### Back up TiDB cluster data
+
+This section describes backup features, impact of backup, and backup file types.
+
+#### Backup features
+
+- **Back up cluster snapshots**: A snapshot of a TiDB cluster contains only the latest and transactionally consistent data at a specific time. You can back up snapshot data of a TiDB cluster using BR. For details, see [Back up TiDB cluster snapshots](/br/br-usage-backup.md#back-up-tidb-cluster-snapshots).
+- **Back up incremental data**: Incremental data of a TiDB cluster is differentiated data between the latest snapshot and the previous snapshot. Incremental data is smaller compared with full data and it is a supplementary to snapshot backup, which reduces the volume of backup data. For details, see [Back up incremental data](/br/br-usage-backup.md#back-up-incremental-data).
+- **Back up a database or a table**: In addition to snapshot and incremental data backup, BR supports backing up a database or table and filtering out unnecessary data. For details, see [Back up a database or table](/br/br-usage-backup.md#back-up-a-database-or-a-table).
+- **Encrypt backup data**: BR supports encrypting data at the backup source or the storage end when data is backed up to Amazon S3. You can select an encryption method as required. For details, see [Encrypt backup data](/br/br-usage-backup.md#encrypt-backup-data).
```

Comment:
```suggestion
- **Encrypt backup data**: BR supports backup data encryption and amazon S3 server-side encryption. You can select an encryption method as required. For details, see [Encrypt backup data](/br/br-usage-backup.md#encrypt-backup-data).
```

Reply:
agree with this modification. Maybe we should synchronize the Chinese as well.